src/kernel/emulation/linux/misc/sysctl_machdep.c

```diff
@@ -28,7 +48,25 @@ const struct known_sysctl sysctls_machdep[] = {
 	   : "0" (level))
 #endif
 
-sysctl_handler(handle_brand_string)
+#ifndef setup
+#define setup(value)\
+            unsigned int level = 0;\
+            unsigned int eax = value;\
+            unsigned int ebx;\
+            unsigned int edx;\
+            unsigned int ecx
+#endif
+
+
+inline void copyout_int(int value, char* to_copy, int to_copy_leng)
```

Comment:
`to_copy_leng` is actually supposed to be a pointer:
```suggestion
inline void copyout_int(int value, char* to_copy, int* to_copy_leng)
```

Reply:
Oh, I missed that one. Just a minute, next commit should fix it.